src/main/scala/com/github/drmarjanovic/app/pg/PgArticlesRepo.scala

```diff
@@ -0,0 +1,22 @@
+package com.github.drmarjanovic.app.pg
+
+import com.github.drmarjanovic.app.domain.{Article, ArticlesRepo}
+import io.getquill.context.ZioJdbc._
+import zio.{Has, Task}
+
+import java.io.Closeable
+import javax.sql.DataSource
+
+final case class PgArticlesRepo(pool: DataSource with Closeable) extends ArticlesRepo {
```

Comment:
Let's give it a full name, you're not saving much with `Pg`. Also, feel free to add some tests.